### ISSUE TITLE
[Feature Request] Toggle hidden posts while browsing subreddits

### ISSUE THREAD

**Issue by Quisit0815** [QuantumBadger/RedReader#687]
It would be awesome to have a toggle switch to show hidden posts while browsing a subreddit.
Sometimes I click on a post and quit the app on accident so the post is hidden through the option in the settings. 
Would be cool to have a toggle in the menu so I don't have to go in the menus when that happens.

**Comment by michalsieron** [comment]
Agree, but I wonder if it wouldn't be better to just put those posts in the "Hidden Posts". And probably have preference to show read posts there.
